### PR TITLE
Added entries for all the changes since 3.8.1

### PR DESCRIPTION
**Pull request by softins**
This is the first draft of the ChangeLog additions since 3.8.1, for review and comment.

It was produced while examining the output of `git log --graph master...r3_8_1`

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,6 +1,106 @@
...
+  (contributed by @henkdegroot)
+
+- Client: Improved the "What's this?" text displayed for Buffer Delay (#2232)
+  (contributed by @henkdegroot)
+
+- Client: Accelerator key clash between Settings button and Settings menu corrected (#2248).
+  (contributed by @henkdegroot)
+
+- Client/Server: output from --help and --version is now sent to StdOut instead of StdErr (#2244).
+  (contributed by @hemkdegroot)
```
[the remark this answers is not in the document]
Comment (softins) [reply ChangeLog:49]: > typo

Yup, just found it a moment ago :)

**Comment by softins** [comment]
I have verified that all the users mentioned in this section of the `ChangeLog` are already in the contributors list in `src/util.cpp`.

**Comment by DavidSavinkoff** [comment]
@vibraphon provided a picture of his home built vibraphone for Jamulus. Please update the ChangeLog to give @vibraphon photograph credit for the vibraphone icon.

- Client: Added Vibraphone to list of instruments (#2080, #2158).
  (contributed by @DavidSavinkoff, @softins)

**Review by hoffie — APPROVED** [review]
One minor nit.

Cross-checked completeness via some new tooling (will send a PR shortly):
```
$ ./tools/is-changelog-complete.sh 
Checking if all Github PRs since 3.8.1 are included for 3.8.2...
-> Missing PR #2249 (Added entries for all the changes since 3.8.1)
Checking if all Github Issues since 3.8.1 are included for 3.8.2...
-> Missing Issue #2247 (Prepare Release 3.8.2)
-> Missing Issue #2214 (Accelerator key collision in several languages)
Checking if all PR or Issue references in git log since r3_8_1 are included for 3.8.2 based on ref origin/master...
-> Missing PR #2221 from git log (Minor correction to PR template)
-> Missing PR #1993 from git log (Fix for #1688 - unicode in mixer should wrap on character boundaries)
```

#2214 should either be de-tagged or must be remembered for inclusion in the Changelog.
All other "findings" are not Changelog-worthy, so this is fine.

File: ChangeLog [hunk 2]
```diff
@@ -1,6 +1,106 @@
...
+  (contributed by @henkdegroot)
+
+- Android: Some internal improvements to the experimental Android port buffer handling (#1528, #2237).
+  (contributed by @j-santander, @softins)
+
+- Mac Installer: Corrected minimum OS version number for Legacy installer from 10.13 to 10.10.
+  Legacy installer will now install correctly on macOS Yosemite or newer (#2223).
+  (contributed by @softins)
+
+- Windows Installer: update NSIS to v3.08 (#2208).
```
[the remark this answers is not in the document]
Comment (softins) [reply ChangeLog]: Fixed

**Comment by softins** [comment]
@hoffie, thanks!

> -> Missing Issue #2214 (Accelerator key collision in several languages)

> #2214 should either be de-tagged or must be remembered for inclusion in the Changelog.

#2214 can only be addressed after the translation cycle. #2248 addressed it for English.

**Comment by softins** [comment]
> ```
> -> Missing PR #1993 from git log (Fix for #1688 - unicode in mixer should wrap on character boundaries)
> ```

Why is #1993 listed here? It was tagged for 3.8.1, not 3.8.2, but was actually closed, not merged, and replaced with #1994. Just thought it might be something to check in your tools.

**Review by pljones — APPROVED** [review]

File: ChangeLog [hunk 3]
```diff
@@ -1,6 +1,106 @@
...
+  (contributed by @DavidSavinkoff, @softins, photo by @vibraphon)
+
+- Client: On Windows, if no driver found while installing, the "Run Jamulus"
+  option will not be checked (#2103).
+  (contributed by @henkdegroot)
+
+- Client: Fix wrong display of Unicode characters at line wrap (#1994).
+  (contributed by @djfun, @pljones)
+
+- Client: Added selection option for level meter style (#1688).
```
Comment (gilgongo) [inline ChangeLog:14]: Can we put this at the top as it's a new feature?

File: ChangeLog [hunk 4]
```diff
@@ -1,6 +1,106 @@
 
-### 3.8.1dev <- NOTE: the release version number will be 3.9.0 ###
+### 3.8.1dev <- NOTE: the release version number will be 3.8.2 ###
 
+- Client: Added Vibraphone to list of instruments (#2043, #2080, #2158).
+  (contributed by @DavidSavinkoff, @softins, photo by @vibraphon)
+
+- Client: On Windows, if no driver found while installing, the "Run Jamulus"
```
Comment (gilgongo) [inline ChangeLog:7]: Seeing as this is an installer thing, can we also put the Mac client install fix below this? It's the main thing for Mac users in this release.

Comment (gilgongo) [reply ChangeLog:7]: Oh wait it's merged - I'll do it myself :-)

**Pull request merged**